notes/templates/notes/topic_form.html

```diff
@@ -0,0 +1,32 @@
+{% extends 'main.html' %}
+
+{% block content %}
+<main class="formPage my-xl">
+    <div class="content-box">
+        <div class="formWrapper">
+            <a class="backButton" href="{{request.GET.next}}"><i class="im im-angle-left"></i></a>
+            <br>
+            {% if page == 'update' %}
+            <h2>Update existing topic</h2>
+            {% else %}
+            <h2>Create new topic</h2>
+            {% endif %}
+
+
+            <form class="form" method="POST" action="">
+                {% csrf_token %}
+
+                {% for field in form %}
+                <div class="form__field">
+                    <label for="formInput#text"> {{field.label}}</label>
```

Comment:
Nie ma id formInput#text. Wartość atrybutu for elementu label powinna mieć wartość id elelemtu, którego label dotyczy. Jeżeli nie ma takiego id to label wyświetli się przy elemencie, przy którym został umieszczony. Czyli for tutaj jest zbędne (i ma niepoprawną wartość).